.gitignore

```diff
@@ -9,6 +9,7 @@ build/
 npm-debug.log
 node_modules/
 src/js/config/Config.dev.ts
+src/js/config/Config.dev.js
```

Comment:
Unnecessary

Reply:
I added because more than once I ended up committing that file by accident. Also I got external contributors to come to ask me about what to do with this file. On my setup, I usually use older clusters in the same branch, and I have to have both files.

Do you think this is enough reason to keep it?

Reply:
doesn't hurt in having it added.